pkg/tests/ossm/yaml/smcp_v2.1.yaml

```diff
@@ -22,3 +22,8 @@ spec:
       enabled: true
   telemetry:
     type: Istiod
+  {{ if eq .ROSA "true"}} 
```

Comment:
This doesn't work, because you pass in `Rosa`, not `ROSA`.

Reply:
If you change the string to a boolean, as I suggested above, you can then simply do:
```suggestion
  {{ if .ROSA }} 
```

Reply:
Yes I can do this, but I have to change the map because is [string]string